internal/config/config.go

```diff
@@ -100,6 +100,7 @@ type Config struct {
 	SyncUsageFile   bool       `yaml:"sync_usage_file,omitempty" ignored:"true"`
 	Fields          []string   `yaml:"fields,omitempty" ignored:"true"`
 	CompareTo       string
+	ChangeTarget    *string
```

Comment:
It seems we have `git-diff`, `ChangeTarget` and `ChangeBase` which (I think) all mean the same thing. Could we make the naming more consistent?

Reply:
yeah, I think these all got mixed up when I was playing around with naming. I think `GitDiffTarget` is the best. So i'll change to that